packages/ui/src/themes/html/RangeSlider.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { forwardRef } from "react";
+import { RangeSliderProps } from "../../theming/ThemeType";
+import * as styles from "./RangeSlider.module.css";
+
+export const RangeSlider = forwardRef<HTMLInputElement, RangeSliderProps>(
+	({ children, ...props }, ref) => {
```

Comment:
You can destructure the 'labels' here, so that you van apply all props on input in one go
```suggestion
	({ children, labels, ...props }, ref) => {
```